src/main/java/org/broadinstitute/hellbender/tools/walkers/varianteval/evaluators/PVariantEvaluator.java

```diff
@@ -0,0 +1,78 @@
+package org.broadinstitute.hellbender.tools.walkers.varianteval.evaluators;
+
+import htsjdk.variant.variantcontext.Genotype;
+import htsjdk.variant.variantcontext.VariantContext;
+import org.broadinstitute.hellbender.engine.FeatureContext;
+import org.broadinstitute.hellbender.engine.ReadsContext;
+import org.broadinstitute.hellbender.engine.ReferenceContext;
+import org.broadinstitute.hellbender.exceptions.UserException;
+import org.broadinstitute.hellbender.tools.walkers.varianteval.util.Analysis;
+import org.broadinstitute.hellbender.tools.walkers.varianteval.util.DataPoint;
+
+@Analysis(description = "Computes different estimates of theta based on variant sites and genotypes")
+
+public class PVariantEvaluator extends VariantEvaluator {
+    @DataPoint(description = "Average variant allele fraction over all variant sites", format = "%.8f")
+    public double avgVarAlleles= 0.0;
+    @DataPoint(description = "Number of called sites over all variant sites;", format = "%d")
+    public int totalCalledSites;
+    @DataPoint(description = "Number of called heterozygous sites;", format = "%d")
+    public int totalHetSites;
+    @DataPoint(description = "Number of called heterozygous sites;", format = "%d")
+    public int totalHomVarSites;
+    @DataPoint(description = "Number of called heterozygous sites;", format = "%d")
+    public int totalHomRefSites;
+
+    final private double ploidy = 2.0; // assume ploidy of 2
+    private double sumVariantAFs; // this is the allele fraction we're summing over all sites, to be used to calculate the avgVarAlleles
```

Comment:
I would rename `sumVariantAFs` to `sumVariantAlleles` because this variable is a count, not fraction

Reply:
But it is a fraction, it's summing the AFs, not counting the alleles

Reply:
gotcha